Terminal.Gui/Input/Keyboard.cs

```diff
@@ -0,0 +1,636 @@
+	/// The key code for the user pressing the space bar
+	/// </summary>
+	Space = 32,
+
+	/// <summary>
+	/// Digit 0.
+	/// </summary>
+	D0 = 48,
+	/// <summary>
+	/// Digit 1.
+	/// </summary>
+	D1,
+	/// <summary>
+	/// Digit 2.
+	/// </summary>
+	D2,
+	/// <summary>
+	/// Digit 3.
+	/// </summary>
+	D3,
+	/// <summary>
+	/// Digit 4.
+	/// </summary>
+	D4,
+	/// <summary>
+	/// Digit 5.
+	/// </summary>
+	D5,
+	/// <summary>
+	/// Digit 6.
+	/// </summary>
+	D6,
+	/// <summary>
+	/// Digit 7.
+	/// </summary>
+	D7,
+	/// <summary>
+	/// Digit 8.
+	/// </summary>
+	D8,
+	/// <summary>
+	/// Digit 9.
+	/// </summary>
+	D9,
+
+	/// <summary>
+	/// The key code for the user pressing Shift-A
+	/// </summary>
+	A = 65,
+	/// <summary>
+	/// The key code for the user pressing Shift-B
+	/// </summary>
+	B,
+	/// <summary>
+	/// The key code for the user pressing Shift-C
+	/// </summary>
+	C,
+	/// <summary>
+	/// The key code for the user pressing Shift-D
+	/// </summary>
+	D,
+	/// <summary>
+	/// The key code for the user pressing Shift-E
+	/// </summary>
+	E,
+	/// <summary>
+	/// The key code for the user pressing Shift-F
+	/// </summary>
+	F,
+	/// <summary>
+	/// The key code for the user pressing Shift-G
+	/// </summary>
+	G,
+	/// <summary>
+	/// The key code for the user pressing Shift-H
+	/// </summary>
+	H,
+	/// <summary>
+	/// The key code for the user pressing Shift-I
+	/// </summary>
+	I,
+	/// <summary>
+	/// The key code for the user pressing Shift-J
+	/// </summary>
+	J,
+	/// <summary>
+	/// The key code for the user pressing Shift-K
+	/// </summary>
+	K,
+	/// <summary>
+	/// The key code for the user pressing Shift-L
+	/// </summary>
+	L,
+	/// <summary>
+	/// The key code for the user pressing Shift-M
+	/// </summary>
+	M,
+	/// <summary>
+	/// The key code for the user pressing Shift-N
+	/// </summary>
+	N,
+	/// <summary>
+	/// The key code for the user pressing Shift-O
+	/// </summary>
+	O,
+	/// <summary>
+	/// The key code for the user pressing Shift-P
+	/// </summary>
+	P,
+	/// <summary>
+	/// The key code for the user pressing Shift-Q
+	/// </summary>
+	Q,
+	/// <summary>
+	/// The key code for the user pressing Shift-R
+	/// </summary>
+	R,
+	/// <summary>
+	/// The key code for the user pressing Shift-S
+	/// </summary>
+	S,
+	/// <summary>
+	/// The key code for the user pressing Shift-T
+	/// </summary>
+	T,
+	/// <summary>
+	/// The key code for the user pressing Shift-U
+	/// </summary>
+	U,
+	/// <summary>
+	/// The key code for the user pressing Shift-V
+	/// </summary>
+	V,
+	/// <summary>
+	/// The key code for the user pressing Shift-W
+	/// </summary>
+	W,
+	/// <summary>
+	/// The key code for the user pressing Shift-X
+	/// </summary>
+	X,
+	/// <summary>
+	/// The key code for the user pressing Shift-Y
+	/// </summary>
+	Y,
+	/// <summary>
+	/// The key code for the user pressing Shift-Z
+	/// </summary>
+	Z,
+	/// <summary>
+	/// The key code for the user pressing A
+	/// </summary>
+	a = 97,
```

Comment:
Don't forget to remove the lower cases letters. In the cases where they are used replaces with 'lowerCase'. e.g. If a key is configured as `Key.a | Key.AltMask` replace with `'a' | Key.AltMask`. Thanks.